src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -86,42 +101,75 @@ def __getitem__(self, key):
             return entity_instance(self.wrapped_data.by_guid(str(key)))
 
     def by_id(self, id):
-        """
-        Return IFC objects filtered by IFC ID.
+        """Return IFC objects filtered by IFC ID.
 
-        Returned objects are unwrapped IFC objects.
+        :param id: STEP numerical identifier
+        :type id: int
+        :returns: Returned objects are unwrapped IFC objects.
+        :rtype: ifcopenshell.entity_instance.entity_instance
         """
         return self[id]
 
     def by_guid(self, guid):
-        """
-        Return IFC objects filtered by IFC GUID.
+        """Return IFC objects filtered by IFC GUID.
```

Comment:
This returns *a single object/instance*, no?

Also what's up with `Returned objects are unwrapped IFC objects`? Is that a bug that needs to be fixed?

Reply:
Fixed :)

No idea why I wrote that ;) Removed.